src/raqm.c

```diff
@@ -1628,14 +1829,19 @@ _raqm_u32_to_u8_index (raqm_t   *rq,
                                        output);
 
   free (output);
+
   return length;
+#endif
 }
 
 /* Convert index from UTF-8 to UTF-32 */
 static uint32_t
 _raqm_u8_to_u32_index (raqm_t   *rq,
                        uint32_t  index)
 {
+#ifdef RAQM_SHEENBIDI
+  return _raqm_count_utf8_codepoints(rq->text_utf8, index);
```

Comment:
Fold the new function into this one and replace the old code entirely.

Reply:
fixed in d0cab8911df86a7af38111fa7fecda13cd6326b7
(I also replaced the function above this one in 61224da0c4d2d55ec0f23540cc6c794f16e4d724)